modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/filters/SelectionFilter.js

```diff
@@ -92,18 +93,21 @@ function SelectionFilter({
 	id,
 	inputPlaceholder,
 	itemKey,
-	itemLabel: itemLabelProp,
+	itemLabel,
 	items: initialItems,
 	multiple,
 	selectedData,
 	setFilter,
 }) {
-	const [query, setQuery] = useState('');
 	const [search, setSearch] = useState('');
 	const [selectedItems, setSelectedItems] = useState(
 		selectedData?.selectedItems || []
 	);
 	const [items, setItems] = useState(apiURL ? null : initialItems);
+	const [localItems, setLocalItems] = useState(
+		initialItems.length ? initialItems : null
```

Comment:
What would it look like if the default value was an empty array(`[]`) instead of null? This way we wouldn't need all the additional checks like `localItems?`

Reply:
Agree with this approach to avoid the `?`